src/rp-paradigm/src/index.js

```diff
@@ -0,0 +1,62 @@
+const rxjs = require('rxjs');
+const { fromEvent, BehaviorSubject } = rxjs;
+
+const generateTodoList = (todos) =>
```

Comment:
I think observing changes in "todos"-argument is a better idea, and then calling the function on its change.

Reply:
I'm calling `rerender function`. Yes, i agree to your argument, but there as we don't have any component, i've used just `rerender`. In the future, it could be changed to `rerenderSomePartOfYourUI()`.